runtime/doc/eval.txt

```diff
@@ -10826,7 +10826,10 @@ win_getid([{win} [, {tab}]])				*win_getid()*
 
 win_gettype([{nr}])					*win_gettype()*
 		Return the type of the window:
+			"aucmdwin"	autocommand window. Temporary window
+					used to execute autocommands.
```

Comment:
Thanks for addition but... the types are prefix for window (popup window, preview window, command(-line) window, unknown window) so I'm afraid aucmd or autocmd are natural.